tests/e2e/tests/mixer/mixer_test.go

```diff
@@ -323,53 +293,37 @@ func TestRateLimit(t *testing.T) {
 		}
 	}()
 
```

Comment:
might be worth commenting what's in rateLimitRule / what we expect to happen here